### ISSUE TITLE
Using tficf without target Y

### ISSUE THREAD

**Issue by banyous** [textvec/textvec#15]
I want to get text vectorization using TfIcfVectorizer() without the need to use Y label vector, Is it possible ? 

tficf = vectorizers.TfIcfVectorizer()
tficf_train = tficf.fit_transform(cv_train, newsgroups_train.target) # here i should give the vectorizer one argument instead of two
tficf_test = tficf.transform(cv_test)

**Comment by zveryansky** [comment]
TfIcf is a supervised vectorization, so you must provide labels.
